src/components/Flex/Flex.scss

```diff
@@ -21,7 +21,8 @@ $o-flex-gutter-width: 0.8rem !default;
 }
 
 .o-flex--vertical-center {
-	align-items: center;
+	flex-direction: column;
+	justify-content: center;
 }
```

Comment:
Dit is best een aparte variant; `align-items` is wel degelijk een verticale centrering. https://codepen.io/ian-emsens-shd/pen/bGvwbzW